cmd/openshift-tests/openshift-tests.go

```diff
@@ -201,6 +202,56 @@ func newRunUpgradeCommand() *cobra.Command {
 	return cmd
 }
 
+func newRunDisasterRecoveryRestoreSnapshotCommand() *cobra.Command {
+	disasterRecoveryOpt := &DisasterRecoveryOptions{}
+	opt := &testginkgo.Options{Suites: append(staticSuites, disasterRecoverySuites...)}
+
+	cmd := &cobra.Command{
+		Use:   "run-dr SUITE",
+		Short: "Run disaster recovery suites",
+		Long: templates.LongDesc(`
+		Help, I'm being held at long description factory
+
+		`) + testginkgo.SuitesString(opt.Suites, "\n\nAvailable test suites:\n\n"),
+
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return mirrorToFile(opt, func() error {
+				if len(args) > 0 {
+					for _, suite := range opt.Suites {
+						if suite.Name == args[0] {
+							disasterRecoveryOpt.Suite = suite.Name
```

Comment:
I don't know why you need this?

Reply:
This would ensure passed suite is one of the static ones and initialize it

Reply:
> I don't know why you need this?

They have skip rules like "no crashloop pods" and the "one alert only".  Is there a different way for him to indicate the suit should skip?  



> This would ensure passed suite is one of the static ones and initialize it

I think the question is why it can't be `openshift-tests run disaster-recovery`, but I don't know how to do "all of a normal parallel except these X" with that.